src/components/month/Day.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <div
+    v-if="!hideLeadingAndTrailingDate"
     :id="'day-' + day.dateTimeString.substring(0, 10)"
     class="calendar-month__weekday"
-    :class="{ 'is-droppable': canBeDropped }"
+    :class="{ 
+      'is-droppable': canBeDropped, 
+      'trailing-or-leading':day.isTrailingOrLeadingDate 
```

Comment:
I'll have to remove this line, since otherwise the styles for a hidden day will be applied also when one doesn't want to hide any days.

Instead we should change the test file.